README.md

```diff
@@ -116,7 +118,17 @@ $ brew install buildkit
 
 To build BuildKit from source, see [`.github/CONTRIBUTING.md`](./.github/CONTRIBUTING.md).
 
-### Starting the `buildkitd` daemon:
+### Configuring `buildctl`
+
+#### Color Output Controls
```

Comment:
Could you put this as a separate item somewhere in the bottom. It's an advanced tweak that  users should not read before they understand how to use buildkit itself.

Reply:
done